### ISSUE TITLE
Segmentation Fault (core dumped)

### ISSUE THREAD

**Issue by asp3n-hayley** [scottwilliambeasley/debian-from-scratch#4]
Get this error after compiling gcc-base 4.9 and partial install of libgcc1. Appears when installing multiarch and libc6, then I can't run any dpkg command without getting that error and no other output. Reboot did not help.

**Comment by asp3n-hayley** [comment]
Update: get the error when running any command in the Chroot environment, ls, tar, etc, not just dpkg

**Comment by tcreek** [comment]
Seems like an OS/hardware issue

**Comment by asp3n-hayley** [comment]
> 
> 
> Seems like an OS/hardware issue

Yeah you might be right. I posted this a while ago, when I had this issue I gave up after trying everything. I may try an LFS build again though

**Comment by tcreek** [comment]
Have you built LFS on the same system before?

**Comment by asp3n-hayley** [comment]
> 
> 
> Have you built LFS on the same system before?

Yes actually, I succeeded in building a minimal system a while ago, then promptly lost it after failure to back it up(I didn't use to be very clever :)).
I think what changed this time is I started the Debian from scratch process after having compiled the entire system, not before, as instructed in the documentation. I will try again after this semester is over and initiate the dfs process as soon as the temporary toolchain is complete, rather than afterwards. I'm not sure that was the issue or not, but it's worth a shot. Maybe I'll try a different package manager if that doesn't work, I like pacman from Arch. My ultimate goal is to create an LFS system that can be maintained without manually rebuilding each package and dependency chains every time a new package is released, while preserving stability. Might be too ambitious haha.
